owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/ShareToRemoteOperationResultParser.kt

```diff
@@ -81,7 +81,9 @@ class ShareToRemoteOperationResultParser(private var shareXmlParser: ShareXMLPar
                             share
                         }
 
-                        result.setData(ShareParserResult(ArrayList(resultData)))
+                        if (resultData != null) {
```

Comment:
You could also write this like:

```
resultData?.let {
    result.data = ShareParserResult(ArrayList(it.toMutableList()))
}
```

Reply:
Yes I could, but is it necessary or has it an advantage ?

Reply:
It's not necessary, is just a suggestion